base/number.jl

```diff
@@ -110,3 +110,27 @@ but you can use `factorial(big(n))` to compute the result exactly in arbitrary p
 If `n` is not an `Integer`, `factorial(n)` is equivalent to [`gamma(n+1)`](:func:`gamma(n+1) <gamma>`).
 """
 factorial(x::Number) = gamma(x + 1) # fallback for x not Integer
+
+"""
+    rescale(x, from_min, from_max, to_min=0.0, newmax=1.0)
```

Comment:
it would need to go into the rst, if it were merged before #18588 - if merged after, I'm not sure what's needed (if anything)